app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationActivityV2.kt

```diff
@@ -1900,12 +1932,19 @@ class ConversationActivityV2 : PassphraseRequiredActionBarActivity(), InputBarDe
         // Voice message too short? Warn with toast instead of sending.
         // Note: The 0L check prevents the warning toast being shown when leaving the conversation activity.
         if (voiceMessageDurationMS != 0L && voiceMessageDurationMS < MINIMUM_VOICE_MESSAGE_DURATION_MS) {
-            Toast.makeText(this@ConversationActivityV2, R.string.messageVoiceErrorShort, Toast.LENGTH_LONG).show()
+            Toast.makeText(this@ConversationActivityV2, R.string.messageVoiceErrorShort, Toast.LENGTH_SHORT).show()
             inputBar.voiceMessageDurationMS = 0L
             return
         }
 
-        // Voice message okay? Attempt to send it.
+        // Put up a relevant Toast if we can see any obvious network issues
+        if (!networkAndSessionNodePathIsValid()) {
+            // We could return here - but instead we'll try our best to send - which will fail - but
+            // it's possible that the message will go into the failed to send state and it may retry
+            // & succeed if network connection is restored.
+        }
```

Comment:
So what is the point of the `if` exactly?

Reply:
Haha, fair point - I'll leave the comment but remove the `if`

Reply:
Addressed via https://github.com/oxen-io/session-android/pull/1523/commits/e83b70c707d13ac0c137a0c65bec938c6ee06e98 and https://github.com/oxen-io/session-android/pull/1523/commits/bd9fdc7a92be36174cf968615ee0efe625e8b9b9